charts/mirrorbits-parent/templates/_helpers.tpl

```diff
@@ -40,3 +40,70 @@ Must be 63 chars. maximum.
 {{- define "mirrorbits-parent.pv-secretname" -}}
 {{- printf "%s-%s" (include "mirrorbits-parent.name" . | trunc 43 ) "persistentvolume-secret" -}}
 {{- end -}}
+
+{{/*
+Define the azure file persistent volume claim name (if enabled).
+Must be 63 chars. maximum.
+*/}}
+{{- define "mirrorbits-parent.pvc-name" -}}
+{{- printf "%s-%s" .Release.Name "mirrorbits-parent-data" -}}
+{{- end -}}
+
+{{/*
+This template method checks the current ingress path and fail with a user facing message if needed or return empty.
+Expected argument: map{
+  "currentBackendService": <string>,
+  "rootContext": { },
+}
+ */}}
+{{- define "mirrorbits-parent.validateIngressBackend" -}}
+{{- if eq .currentBackendService "mirrorbits" -}}
+  {{- if not (index .rootContext.Values "mirrorbits-lite" "enabled") -}}
+    {{- fail "Cannot use mirrorbits(-lite) as backend if it is disabled." }}
+  {{- end -}}
+  {{- if not (index .rootContext.Values "mirrorbits-lite" "backendServiceNameTpl") -}}
+    {{- fail "Cannot determine mirrorbits backend service due to missing value 'mirrorbits-lite.currentBackendServiceNameTpl." }}
+  {{- end -}}
+{{- else if eq .currentBackendService "httpd" -}}
+  {{- if not (index .rootContext.Values "httpd" "enabled") -}}
+    {{- fail "Cannot use httpd as backend if it is disabled." }}
+  {{- end -}}
+  {{- if not (index .rootContext.Values "httpd" "backendServiceNameTpl") -}}
+    {{- fail "Cannot determine httpd backend service due to missing value 'httpd.currentBackendServiceNameTpl." }}
+  {{- end -}}
+{{- else -}}
+  {{- fail "Required key: backendService for ingress.hosts[].paths[] objects must have the value 'httpd' or 'mirrorbits'." }}
```

Comment:
```suggestion
  {{- fail "Required key: backendService for ingress.hosts[].paths[] objects must have the value 'httpd' or 'mirrorbits-lite'." }}
```
Same here, not sure, but it could help to be consistent.

Reply:
👎 for this one as it is a user-facing message